### PR TITLE
Added a new feature

### PR DESCRIPTION
**Pull request by KRoos555**
This feature is for adding an autocomplete into the search box.

### CODE REVIEW

**Review by NoodlesCupCake — COMMENTED** [review]
This content isn't well-written, please improve it.

**Review by NoodlesCupCake — COMMENTED** [review]
improve!

**Review by NoodlesCupCake — COMMENTED** [review]
improve!

**Review by NoodlesCupCake — COMMENTED** [review]
good

File: KroosFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class KroosFeature {
+    public void feature() {
+        //this is a new feature
```
[the remark this answers is not in the document]
Comment (NoodlesCupCake) [reply KroosFeature.java]: This content is not well-written. Please improve it.

**Review by NoodlesCupCake — COMMENTED** [review]
This content is not well-written. Please improve it.

**Comment by KRoos555** [comment]
The new fix is already available.

**Review by Pontuspts — COMMENTED** [review]
good

File: KroosFeature.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+public class KroosFeature {
+      public void feature() {
+          // this is a new feature
+          int x = 0;
+          x = x+1;
```
Comment (Vvidsky) [inline KroosFeature.java:5]: Good job!

**Review by Vvidsky — APPROVED** [review]

File: KroosFeature.java (hunk 2 above)
Comment (ChanAndKlee) [reply KroosFeature.java:5]: Good job!

File: KroosFeature.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+public class KroosFeature {
+      public void feature() {
+          // this is a new feature
```
Comment (ChanAndKlee) [inline KroosFeature.java:3]: Nice

**Review by Jack5hj — COMMENTED** [review]
This content is not well-written. Please improve it.

**Pull request merged**